sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -192,6 +193,9 @@ def get_queue_receiver(self, queue_name, **kwargs):
         """Get ServiceBusReceiver for the specific queue.
 
         :param str queue_name: The path of specific Service Bus Queue the client connects to.
+        :keyword Optional[SubQueue] sub_queue: If specified, the subqueue this receiver will connect to.
```

Comment:
nit: do we want the docstring in this format:
```python
:keyword sub_queue:
:paramtype sub_queue: Optional[~azure.servicebus.SubQueue]
```

Reply:
Does it change the doc rendering in any meaningful way?  I can check with sphinx but I'm inclined to make this a note on one of our docstring cleanup bugs for next sprint.  (am still curious what tractable difference it'd make to the reference)

Reply:
(Added to docstring cleanup issue a note to this discussion and steps to investigate impact)